website/pages/playground/index.html

```diff
@@ -359,7 +359,7 @@
         </div>
         <div class="options last">
           <label>--trailing-comma <select id="trailingComma"><option value="none">none</option><option value="es5">es5</option><option value="all">all</option></select></label>
-          <label>--parser <select id="parser"><option value="babylon">babylon</option><option value="flow">flow</option><option value="typescript">typescript</option><option value="postcss">postcss</option><option value="json">json</option><option value="graphql">graphql</option></select></label>
+          <label>--parser <select id="parser"><option value="babylon">babylon</option><option value="flow">flow</option><option value="typescript">typescript</option><option value="postcss">postcss</option><!-- TODO: Enable these when prettier@>1.7.0 has been released. <option value="postcss-less">postcss-less</option><option value="postcss-scss">postcss-scss</option>--><option value="json">json</option><option value="graphql">graphql</option></select></label>
```

Comment:
1.8.0 now

Reply:
Exclusive `>1.7.0` is technically correct :wink: